stores/pallidsturgeonstore.go

```diff
@@ -1106,58 +1108,65 @@ func (s *PallidSturgeonStore) SaveSupplementalDataEntry(supplementalDataEntry mo
 var updateSupplementalDataSql = `UPDATE ds_supplemental SET 
 f_fid = :2,
 mr_id = :3,
-TAGNUMBER = :4, 
-PITRN = :5, 
-SCUTELOC = :6, 
-SCUTENUM = :7, 
-SCUTELOC2  = :8, 
-SCUTENUM2 = :9, 
-ELHV = :10, 
-ELCOLOR = :11, 
-ERHV = :12, 
-ERCOLOR = :13, 
-CWTYN  = :14, 
-DANGLER = :15, 
-genetic = :16, 
-genetics_vial_number = :17,
-BROODSTOCK = :18, 
-HATCH_WILD = :19, 
-head = :20, 
-snouttomouth = :21, 
-inter = :22, 
-mouthwidth = :23, 
-m_ib = :24, 
-l_ob = :25, 
-l_ib = :26, 
-r_ib = :27, 
-r_ob = :28, 
-anal = :29, 
-dorsal = :30, 
-status = :31, 
-HATCHERY_ORIGIN = :32, 
-SEX = :33, 
-stage = :34,  
-recapture = :35, 
-photo = :36, 
-genetic_needs = :37, 
-other_tag_info = :38, 
-comments = :39, 
-f_id = :40,
-edit_initials = :41,
-last_edit_comment = :42,	
-last_updated = :43, 
-uploaded_by = :44,
-species_id = :45,
-complete = :46,
-approved = :47,
-checkby = :48,
-recorder = :49
+netrivermile = :4,
+length = :5,
+weight = :6,
+TAGNUMBER = :8, 
+PITRN = :9, 
+SCUTELOC = :10, 
+SCUTENUM = :11, 
+SCUTELOC2  = :12, 
+SCUTENUM2 = :13, 
+ELHV = :14, 
+ELCOLOR = :15, 
+ERHV = :16, 
+ERCOLOR = :17, 
+CWTYN  = :18, 
+DANGLER = :19, 
+genetic = :20, 
+genetics_vial_number = :21,
+BROODSTOCK = :22, 
+HATCH_WILD = :23, 
+head = :24, 
+snouttomouth = :25, 
+inter = :26, 
+mouthwidth = :27, 
+m_ib = :28, 
+l_ob = :29, 
+l_ib = :30, 
+r_ib = :31, 
+r_ob = :32, 
+anal = :33, 
+dorsal = :34, 
+status = :35, 
+HATCHERY_ORIGIN = :36, 
+SEX = :37, 
+stage = :38,  
+recapture = :39, 
+photo = :40, 
+genetic_needs = :41, 
+other_tag_info = :42, 
+comments = :43, 
+f_id = :44,
+edit_initials = :45,
+last_edit_comment = :46,	
+last_updated = :47, 
+uploaded_by = :48,
+species_id = :49,
+complete = :50,
+approved = :51,
+checkby = :52,
+recorder = :53
 WHERE s_id = :1`
 
 func (s *PallidSturgeonStore) UpdateSupplementalDataEntry(supplementalDataEntry models.UploadSupplemental) error {
 	_, err := s.db.Exec(updateSupplementalDataSql,
 		supplementalDataEntry.FFid,
 		supplementalDataEntry.MrId,
+		supplementalDataEntry.NetRiverMile,

```

Comment:
Netrivermile, length, width, and condition will be read-only fields and will not be updated when updating a supplemental data entry.
So you do not need to update UpdateSupplementalDataEntry.